### ISSUE TITLE
A method for simpler adding "aria-current" to a menu item

### ISSUE THREAD

**Issue by thisbit** [builderius/builderius#71]
I an working on an accessible menu. And I find that adding aria-current dynamically is a bit complex. 

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-current

One idea is to compare url on each menu link to the url of the current page. I had tried that previously,  and remembered I have not had success. 

This is a very common requirement for menus, even just for stylistic purpose, and as such I propose we create a helper function/method to be available in our nav query. We should be able to both add aria-current this way and .current class.
